Policies, General Operations/Meetings_and_Accounting.md

```diff
@@ -1,67 +1,26 @@
 # Meetings
 
-##### Summary
-
-The CCT will have meetings monthly.  This is to encourage participation in decision making and prevent complacency, apathy or maintaining the status quo.  
-
-Meetings will be scheduled in a way that allows maximum participation with the CPM, ACPM and CCT.  However, meetings will not be conducted without a quorum.  A quorum is 50% of the total number of CPM, ACPM and CCT members in the meeting (rounded up).  Meetings will be scheduled for the 2nd or 3rd week of the month, unless that week is not suitable for all members to attend (or at least a quorum), at which time the meeting will be scheduled each week following until at least a quorum is obtained.  The default meeting time and date will be the 2nd Thursday of the month at 9:00 PM Eastern time zone
-
-Note: A new meeting frequency, quorum rules, or scheduling rules can be obtained by ammending this policy per the Policies and Opearations Manual: Policies, General Operations section
-
-##### Meeting agenda
-
-Meetings will discuss, at a minimum:
-1) Old business (including previous policies in presentation phase)
-2) New business (including new policies in presentation phase)
-3) Financial summary
-4) Community quorum
-5) CCT session (CCT, CPM and ACPM only)
-	CCT session will only discuss matters not appropriate for the entire community, specifically those that pertain to a member of the community.
-
-Suggested topics of conversation include (in no particular order):
-- How to reach more users
-- The current state of the project
-- The future of the project
-- Breaches of the policies and operations manual and community guidelines and remedial actions that need to be taken or has been taken
-- Action items that have been resolved, and which need to be removed
-- How to raise more money for the project
-- How money is being spent
-
-Any action in the meeting will have motion and will have a second.  This includes starting the meeting and finishing the meeting.
-
-##### Venue
-
-The venue for the meetings will be discord audio chat for all members.  CCT members are expected to have speakers and microphone available, but it is understood that not all circumstances permit audio meetings.  Therefore, any member can use the chat component if necessary.
-
-If a potential atendee informs the CCT, CPM or ACPM that they do not want to use Discord due to its proprietary nature, the meeting will be held on Mumble.
+Meetings will be held in continuously in the cct-text channel.  CCT members are expected to check the channel and threads and communicate daily as required.  
```

Comment:
Maybe make this clear that this isn't a meeting, but just continuous communication within the CCT, amongst it's members?

Reply:
changed